HHS/Head-Start-TTADP
# Rename grantee to recipient on be

## Description of change

Sorry lots of changes here. Only renamed variables/labels. No added functionality or code, outside of the migration. I'm not expecting this one to be reviewed quickly :upside_down_face: 

This includes renaming all variables. Non-grantee is now other entity/entities. A migration renames tables and constraints as needed. The migrations also change the values of some columns:

 * Reasons of 'New Grantee' are now 'New Recipient'
 * Activity recipient type updated to be 'recipient' or 'other-entity'
 * Requester values of 'grantee' updated to 'recipient'

## How to test

Use the site in dev. Run migrations and roll back locally. Make sure everything still works. Check the data for the three columns above in draft and approved reports. Create new reports from scratch.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-463

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
